BranchSDK-Samples/Windows/TestBed/BranchOperations.cpp

```diff
@@ -40,7 +40,7 @@ BranchOperations::setupSDKLogging()
         (void)_wmkdir(String(branchLogFilePath).wstr().c_str());
     }
     else {
-        // If the %AppData% env. var. is not set for some reason, use the cwd.
+        // If the %LocalAppData% env. var. is not set for some reason, use the cwd.
```

Comment:
Since we are switching to LocalAppData as first choice now, wouldn't AppData be a better 2nd choice than cwd ?

Reply:
I had the same thought. I just didn't want to spend the time on it. This is only TestBed, not the SDK, though maybe we could move this logic into the SDK to generate file-based logs in a standard location. In any case, it seems an unlikely problem that the `AppData` or `LocalAppData` is not set. And if `LocalAppData` is not set, why should I expect `AppData` to be? But cwd is probably not writable in plenty of cases. We could also support an override like`BranchIOLogFolder`. I think for now it's ok, but maybe we can do better in the SDK itself.

Reply:
I was also thinking that this is just TestBed and not a big deal. True that unlikely that the AppData or LocalAppData directories are not set.